### PR TITLE
explore area improvements, remove feature flag (enable)

### PR DESCRIPTION
**Pull request by sqs**
See commit messages.

> This PR updates the CHANGELOG.md file to describe any user-facing changes.

### CODE REVIEW

File: enterprise/src/extensions/explore/ExtensionsExploreSection.tsx [hunk 1]
```diff
@@ -70,26 +70,25 @@ export class ExtensionsExploreSection extends React.PureComponent<Props, State>
...
-                        <div className="card-deck">
-                            {extensionsOrError.slice(0, ExtensionsExploreSection.QUERY_EXTENSIONS_ARG_FIRST).map(
-                                (extension /* or loading */, i) =>
-                                    extension === LOADING ? (
+                        <div className="row">
+                            {extensionsOrError.slice(0, ExtensionsExploreSection.QUERY_EXTENSIONS_ARG_FIRST).map((
+                                extension /* or loading */,
+                                i
+                            ) => (
+                                <div key={i} className="col-md-6 col-lg-3 mb-2">
```
Comment (felixfbecker) [inline enterprise/src/extensions/explore/ExtensionsExploreSection.tsx]: Wouldn't the extension ID be a better key?

File: enterprise/src/extensions/explore/ExtensionsExploreSectionExtensionCard.tsx [hunk 2]
```diff
@@ -9,7 +9,12 @@ export const ExtensionsExploreSectionExtensionCard: React.SFC<{
     description?: string | React.ReactFragment
     url?: string
 }> = ({ title, description = '', url }) => (
-    <LinkOrSpan to={url} className="card bg-secondary border-primary card-link text-white">
+    <LinkOrSpan
+        to={url}
+        className="card bg-secondary border-primary card-link text-white"
+        // tslint:disable-next-line:jsx-ban-props
```
Comment (felixfbecker) [inline enterprise/src/extensions/explore/ExtensionsExploreSectionExtensionCard.tsx]: What's the justification?

File: enterprise/src/extensions/explore/ExtensionsExploreSection.tsx (hunk 1 above)
Comment (sqs) [reply enterprise/src/extensions/explore/ExtensionsExploreSection.tsx]: In this case, `extension` is `GQL.IRegistryExtension | typeof LOADING`, and in the latter case there is no extension ID (yet). So we need to use `i`.

(We could rename the variable to `extensionOrLoading`, but IMO that reduces readability overall.)

File: enterprise/src/extensions/explore/ExtensionsExploreSectionExtensionCard.tsx (hunk 2 above)
Comment (sqs) [reply enterprise/src/extensions/explore/ExtensionsExploreSectionExtensionCard.tsx]: Added the justification (that these are custom styles that we will move into Bootstrap soon, but we haven't done that yet). For this one-off thing, given that it's entirely a presentation concern, it didn't seem worthwhile to make a separate stylesheet.

**Pull request merged**